cli:  Add get-epoch-info command

mvines opened the pull request:
```
$ solana get-epoch-info
Keypair: /Users/mvines/validator-keypair.json
RPC Endpoint: http://tds.solana.com:8899

Current epoch: 13
Total slots in current epoch: 8192
Remaining slots in current epoch: 995
Time remaining in current epoch: 6.63 minutes
```

mvines commented:
I didn't add it cause `solana get-slot` already provides the abs slot.  That seem ok?

CriesofCarrots commented:
> I didn't add it cause `solana get-slot` already provides the abs slot. That seem ok?

I don't have a real strong opinion, but that same argument was true about the RPC endpoints (getEpochInfo and getSlot) as well.

mvines commented:
Ok I added the current slot and cleaned up the time formatting a little:
```
$ solana -u http://127.0.0.1:8899 get-epoch-info
Keypair: /Users/mvines/validator-keypair.json
RPC Endpoint: http://127.0.0.1:8899

Current epoch: 3
Current slot: 242
Total slots in current epoch: 256
Remaining slots in current epoch: 238
Time remaining in current epoch: 1 minutes, 35 seconds
```
